grpcutil/jwt_auth.go

```diff
@@ -108,3 +122,40 @@ func (g *JWTAuthenticationHandler) RequestPrincipal(ctx context.Context) (string
 
 	return principal, nil
 }
+
+type HandlerOption func(handler *JWTAuthenticationHandler)
```

Comment:
The option pattern usually _returns_ a pointer too. That gives opts the flexibility to possibly return an entirely new object.